netci.groovy

```diff
@@ -32,20 +32,43 @@ def static getOSGroup(def os) {
 
 // We use this class (vs variables) so that the static functions can access data here.
 class Constants {
+
     // Innerloop build OS's
     // The Windows_NT_BuildOnly OS is a way to speed up the Non-NT builds temporarily by avoiding
     // test execution in the build flow runs.  It generates the exact same build
     // as Windows_NT but without the tests.
-    def static osList = ['Ubuntu', 'Debian8.4', 'OSX', 'Windows_NT', 'Windows_NT_BuildOnly', 'FreeBSD', 'CentOS7.1', 'OpenSUSE13.2', 'OpenSUSE42.1', 'RHEL7.2', 'LinuxARMEmulator', 'Ubuntu16.04', 'Ubuntu16.10', 'Fedora23']
+    def static osList = [
+               'Ubuntu',
+               'Debian8.4',
+               'OSX',
+               'Windows_NT',
+               'Windows_NT_BuildOnly',
+               'FreeBSD',
+               'CentOS7.1',
+               'OpenSUSE13.2',
+               'OpenSUSE42.1',
+               'RHEL7.2',
+               'LinuxARMEmulator',
+               'Ubuntu16.04',
+               'Ubuntu16.10',
+               'Fedora23']
+
     def static crossList = ['Ubuntu', 'OSX', 'CentOS7.1', 'RHEL7.2', 'Debian8.4', 'OpenSUSE13.2']
+
     // This is a set of JIT stress modes combined with the set of variables that
     // need to be set to actually enable that stress mode.  The key of the map is the stress mode and
     // the values are the environment variables
-    def static jitStressModeScenarios = ['minopts' : ['COMPlus_JITMinOpts' : '1'], 'forcerelocs' : ['COMPlus_ForceRelocs' : '1'],
-               'jitstress1' : ['COMPlus_JitStress' : '1'], 'jitstress2' : ['COMPlus_JitStress' : '2'],
-               'jitstressregs1' : ['COMPlus_JitStressRegs' : '1'], 'jitstressregs2' : ['COMPlus_JitStressRegs' : '2'],
-               'jitstressregs3' : ['COMPlus_JitStressRegs' : '3'], 'jitstressregs4' : ['COMPlus_JitStressRegs' : '4'],
-               'jitstressregs8' : ['COMPlus_JitStressRegs' : '8'], 'jitstressregs0x10' : ['COMPlus_JitStressRegs' : '0x10'],
+    def static jitStressModeScenarios = [
+               'minopts' : ['COMPlus_JITMinOpts' : '1'],
```

Comment:
Funny I had this same reformatting in my pending change.  Oh well...probably not too hard to merge.